specification/web/resource-manager/Microsoft.Web/stable/2024-11-01/WebApps.json

```diff
@@ -25738,6 +25738,29 @@
         }
       }
     },
+    "ErrorPage": {
+      "description": "A custom error page for a specific status returned by a web app.",
+      "type": "object",
+      "properties": {
+        "statusCode": {
+          "description": "The status code for which the error page will be used",
+          "type": "integer",
+          "format": "int32"
+        },
+        "content": {
+          "description": "The content of the error page",
+          "type": "string"
+        },
+        "contentType": {
```

Comment:
>contentType [](http://example.com/codeflow?start=9&length=11)

what possible values can this property have?

Reply:
Updated